lib/generators/paper_trail/install_generator.rb

```diff
@@ -31,6 +32,13 @@ def create_migration_file
       end
     end
 
+    def create_initializer
+      create_file(
```

Comment:
Looks like the deprecation warning is important for people who previously were tracking associations solely by the presence of the `version_associations` table.  See #785 for reference